libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -115,7 +109,9 @@ protected synchronized void upgradeAccMgr() {
             	upgradeTo2Dot2();
             }
         } catch (NumberFormatException e) {
-        	Log.e("UpgradeManager:upgradeAccMgr", "Failed to parse installed version.");
+            SalesforceLogger.getLogger(SalesforceSDKManager.SF_SDK_COMPONENT_NAME,
```

Comment:
I'm not sure about the class name but it should be as short as possible and live in the SalesforceSDK lib.